varnish-mixin/panels.libsonnet

```diff
@@ -0,0 +1,267 @@
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Sessions rate stat
+      sessionsRatePanel:
+        commonlib.panels.generic.stat.base.new(
+          'Sessions rate',
+          targets=[
+            signals.sessions.sessionsRate.asTarget(),
+          ],
+          description='The rate of total sessions created in the Varnish Cache instance.'
+        )
+        + g.panel.stat.standardOptions.withUnit('/ sec')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Cache hits stat
+      cacheHitsPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Cache hits',
+          targets=[
+            signals.cache.cacheHits.asTarget(),
+          ],
+          description='The rate of cache hits.'
+        )
+        + g.panel.stat.standardOptions.withUnit('/ sec')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Cache hit pass stat
+      cacheHitPassPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Cache hit pass',
+          targets=[
+            signals.cache.cacheHitPass.asTarget(),
+          ],
+          description='Rate of cache hits for pass objects (fulfilled requests that are not cached).'
+        )
+        + g.panel.stat.standardOptions.withUnit('/ sec')
+        + g.panel.stat.panelOptions.withTransparent(true),
+
+      // Session queue length stat
+      sessionQueueLengthPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Session queue length',
+          targets=[
+            signals.sessions.sessionQueueLength.asTarget(),
+          ],
+          description='Length of session queue waiting for threads.'
+        )
+        + g.panel.stat.standardOptions.withUnit('none')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Thread pools stat
+      poolsPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Pools',
+          targets=[
+            signals.threads.threadPools.asTarget(),
+          ],
+          description='Number of thread pools.'
+        )
+        + g.panel.stat.standardOptions.withUnit('none')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Backend connections timeseries
+      backendConnectionsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Backend connections',
+          targets=[
+            signals.backend.backendConnectionsAccepted.asTarget(),
+            signals.backend.backendConnectionsRecycled.asTarget(),
+            signals.backend.backendConnectionsReused.asTarget(),
+            signals.backend.backendConnectionsBusy.asTarget(),
+            signals.backend.backendConnectionsUnhealthy.asTarget(),
```

Comment:
All these signals previously used the `irate()` function and I'm not quite sure how useful it is to know rate per second of the connection... I've currently translated as originally implemented but it seemed like we should maybe be displaying using `increase`. Happy to hear other thoughts, but just figured that knowing a whole number connection value here would be more user friendly/intuitive.